### ISSUE TITLE
🚀 Feature: Add missing accessibility features

### ISSUE THREAD

**Issue by arjunr00** [UCLA-Creative-Labs/sunshine#51]
[Here's a link to our Figma design.](https://www.figma.com/file/dm5sWSjkqeooXCFTDqrpAf/FINAL-Website?node-id=420%3A2)

### Use Case

We gotta support our differently-abled peers.

### Proposed Solution

* [x] :wave: I may be able to implement this feature request

---

This is a :rocket: Feature Request
